### PR TITLE
Add `copy-*` icons

### PR DESCRIPTION
**Pull request by danielbayley**
or `close-all`…

### CODE REVIEW

File: icons/x-square-all.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="13" height="13" x="9" y="9" rx="2" ry="2" />
+  <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1" />
+  <line x1="18" x2="13" y1="13" y2="18" />
+  <line x1="13" x2="18" y1="13" y2="18" />
```
Comment (jguddas) [inline icons/x-square-all.svg:15]:
```suggestion
  <rect x="9" y="9" width="13" height="13" rx="2" ry="2"/>
  <path d="m13 13 5 5"/>
  <path d="m18 13-5 5"/>
  <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1"/>
```

**Review by ericfennis — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
Hm... since this is exactly the same icon as `copy`, just with an added `x`, maybe it should be named `copy-x` instead?

And while we're there, maybe `copy-plus`, `copy-minus` and `copy-check` could also be useful? (not totally sure, but maybe for some clipboard magic or diff actions).
(for the latter, the base icon would have to be modified to be of an even numbered size see below)
image

File: icons/copy.svg [hunk 2]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="9" y="9" width="13" height="13" rx="2" ry="2" />
-  <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1" />
+  <path d="M4 16c-1.1 0-2-.9-2-2V4c0-1.1.9-2 2-2h10c1.1 0 2 .9 2 2" />
```
Comment (ericfennis) [inline icons/copy.svg:12]: I'm not sure with this change. there is more code added.
I think we can better use the old rect and path.

File: icons/copy-equal.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg
```
Comment (ericfennis) [inline icons/copy-equal.svg:1]: What is the use case exactly of this icon?

File: icons/copy.svg (hunk 2 above)
Comment (danielbayley) [reply icons/copy.svg:12]: Yeah, Illustrator seems reluctant to output `rect`s. I’ll just code these manually instead…

File: icons/copy-equal.svg (hunk 3 above)
Comment (danielbayley) [reply icons/copy-equal.svg:1]: I was just aiming for all `circle-*`, `square-*` and `copy-*` (essentially just multiple squares) icons to be more of a set with the same set of elements…

Comment (ericfennis) [reply icons/copy-equal.svg:1]: Ah check, I get your thinking.
But I'm not sure about the use case. Let's keep this one out if we couldn't come up with a use case.

**Comment by karsa-mistmere** [comment]
@danielbayley There's a conflict from concurrent optimization of icon files, can you resolve it, then I think this is also good to go, great work. :)

**Review by ericfennis — APPROVED** [review]
Awesome work!

**Pull request merged**